### PR TITLE
Automatically install role dependencies

### PR DESCRIPTION
**Pull request by bborysenko**
This is PR for #71. 

The main goal is to simplify the testing of roles that have some dependencies. For the moment I'm using a simple wrapper around molecule to install extra roles with `ansible-galaxy`. But would like to use molecule only for that.

Changes in brief:

- update `ansible.cfg.j2` template to set `roles_path` to something like `.molecule/roles:../`
- add `requirements_file` option to `ansible` section of `molecule.yml`
- if `requirements_file` is set, then role dependencies will be automatically installed with `ansible-galaxy` during `molecule converge` to first directory specified in `roles_path`.

### CODE REVIEW

**Comment by retr0h** [comment]
:100: 

This is a pretty clean solution.

File: molecule/commands.py [hunk 1]
```diff
@@ -35,6 +35,7 @@
 import molecule.utilities as utilities
 import molecule.validators as validators
 from molecule.ansible_playbook import AnsiblePlaybook
+from molecule.ansible_galaxy_install import AnsibleGalaxyInstall
```
Comment (retr0h) [inline molecule/commands.py]: This should go above `AnsiblePlaybook` (alphabetical imports)

**Comment by bborysenko** [comment]
@retr0h thanks, fixed

**Comment by abrown-sg** [comment]
:+1:

**Pull request merged**